shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/usecase/GetProfileUseCase.kt

```diff
@@ -0,0 +1,20 @@
+package co.nimblehq.blisskmmic.domain.usecase
+
+import co.nimblehq.blisskmmic.domain.model.User
+import co.nimblehq.blisskmmic.domain.repository.UserRepository
+import kotlinx.coroutines.flow.Flow
+
+interface GetProfileUseCase {
+
+    operator fun invoke(): Flow<User>
+}
+
+class GetProfileUseCaseImpl(
+    private val userRepository: UserRepository
+) : GetProfileUseCase {
+
+    override operator fun invoke(): Flow<User> {
+        return userRepository
+            .getProfile()
+    }
```

Comment:
A small suggestion 🙏
```suggestion
    override operator fun invoke() = userRepository.getProfile()
```